examples/common/avb.c

```diff
@@ -72,10 +72,17 @@ int pci_connect(device_t * igb_dev)
 
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
-		if (err) {
+		if (err || igb_attach_tx(igb_dev)) {
```

Comment:
It is; that's apparently *also* part of this patch set, though.  It should be the only one; @danjurious, if you remove this chunk and leave the one that adds in the clearer version at line 80, I think it'll be acceptable.